tests/github-action-helper.sh

```diff
@@ -104,6 +104,28 @@ wait_for_operator_pod_to_be_ready_state_custom() {
 EOF
 }
 
+wait_for_command_to_get_success() {
```

Comment:
wait_for_command_to_get_success seems unnecessarily long to me. IMO, wait_for_condition is still clear and not as long.

```suggestion
wait_for_condition() {
```

Reply:
yup